plugins/destination/s3/docs/authentication_advanced.md

```diff
@@ -0,0 +1,79 @@
+---
+hub-title: Advanced Authentication Configuration
```

Comment:
```suggestion
hub-title: Auth Details
```

I believe this is what we used in a another place to keep the sidebar navigation title shorter